Integrations/integration-QRadar.yml

```diff
@@ -455,5 +473,33 @@ script:
     - name: range
       description: Number of results in return
     description: Get closing reasons
+  - name: qradar-create-note
+    arguments:
+    - name: offense_id
+      required: true
+      default: true
```

Comment:
Does it make sense to make an arg default if we have more than one arg required? You will have to use arg names in the cli anyways

Reply:
I think this is ok

Reply:
it will just b the first one to pop up